src/reference/antora/.github/workflows/deploy-docs.yml

```diff
@@ -0,0 +1,33 @@
+name: Deploy Docs
```

Comment:
I think this GHA workflow file is some kind of duplication and out of use.
Or is that a new approach how things go from now on?
Then why do we need it then in the top-level `.github/workflows`?..